src/components/SortControl.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import {Meta, StoryObj} from "@storybook/react";
+import {action} from "@storybook/addon-actions";
+import SortControl from "./SortControl";
+
+const meta = {
+    component: SortControl,
+    title: 'SortControl',
+    tags: ['autodocs'],
+    argTypes: {
+        currentSelection: {
+            control: {type: "select"},
+            options: ["Release Date", "Title"],
+        }
+    },
+    args: {
+        currentSelection: "Release Date",
+    }
+} satisfies Meta<typeof SortControl>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+    args: {
+        currentSelection: "Release Date",
```

Comment:
Looks good! Forgot to mentioned that would be better to improve hierarchy of the folders in the project.  
Something like this:
![image](https://github.com/user-attachments/assets/8f3dc663-3688-43e7-9ea0-d1e06c9be60c)
hooks folder for hooks, component => InputFolder => all files to Input component(styles, jsx, maybe story for the storybook as well), shared => shared can include components folder as well -> we usually put to shared those that repeated in all deferent places. Do not need to fix now, but fix please in the task. If you have any question ask it in the near call.